src/containers/community/CommunitySettings.js

```diff
@@ -181,6 +187,13 @@ export default class CommunitySettings extends React.Component {
       }
     }
     this.setState({expand: {...expand, [section]: open || !expand[section]}})
+    if (goToJoinRequests) {
+      setTimeout(() => {
```

Comment:
I think you can check for the change in componentDidUpdate. setTimeout is fine too if it works though